Dockerfile.lab

```diff
@@ -12,6 +12,7 @@ RUN yarn install --frozen-lockfile
 ENV NEXT_PUBLIC_ENABLE_STANDALONE_OUTPUT=true
 ENV NEXT_PUBLIC_ENABLE_EXPLORER=true
 ENV NEXT_PUBLIC_DEFAULT_NETWORK=custom
+ENV NEXT_PUBLIC_RESOURCE_PATH=/lab
```

Comment:
What does it do/fix?

Reply:
it fixes the broken images reported here: https://github.com/stellar/laboratory/issues/1619

This requires an accompanying change that @quietbits is working on at https://github.com/stellar/laboratory/tree/public-asset-path